modules/core/src/read-file/read-file.js

```diff
@@ -48,16 +48,16 @@ export function readFile(uri, options = {}) {
       readFileObject(uri, options);
     }
 
+    if (typeof URL !== 'undefined') {
```

Comment:
Please add comment explaining why this check is needed?

So far my understanding was that `URL` is present on all three environments (main thread/worker/node)....

Perhaps this was just an artifact of the image code still being a WIP and not being fully cleaned/up and aligned with the new `images` submodule.

Reply:
`URL` does not exist in the global scope under Node.

Reply:
> `URL` does not exist in the global scope under Node.

See https://nodejs.org/api/url.html, in particular https://nodejs.org/api/url.html#url_the_whatwg_url_api.

Availability may depend on version etc, but I believe I was successfully using it myself somewhere else in loaders.gl just recently.

Reply:
It's only made available in Node 10. My understanding is that we are only requiring Node 10 for dev, not on the user end?

Reply:
I see. A reason why we need to restore Node 8 CI...

So Node will sometimes hit this path, if version is > 10?